dbld/images/packages/packages.txt

```diff
@@ -0,0 +1,171 @@
+#autogen
+[
+    autoconf                    [centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+    autoconf268                 [centos6]
+]
+autoconf-archive                [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+automake                        [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+ca-certificates                 [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+git                             [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+libtool                         [centos6, centos7, d_jessie, d_stretch, d_buster, u_trusty, u_xenial, u_bionic, u_cosmic]
+[
```

Comment:
could we add support for "*" element here? it could become cumbersome to maintain this list, as any new release would need a new entry to all of the lines.

Reply:
we've discussed this and came to the conclusion that we can use the name of the distro (e.g. ubuntu or centos) without version number to specify the name for a specific distro as a whole.

this is a simple form of wildcarding, and more complex would complicate the file parsing a lot (which is very simple/dumb now).

This should work for now.